src/Components/Nav/components/Category.js

```diff
@@ -0,0 +1,83 @@
+import React, { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+import { API } from '../../../config';
+
+function Category(props) {
+  const [categoryList, setCategoryList] = useState([]);
+
+  useEffect(() => {
+    fetch('/data/categoryData.json')
+      .then(res => res.json())
+      .then(categoryList => setCategoryList(categoryList.category_lists));
+  }, []);
+
+  // useEffect(() => {
+  //   fetch(`${API}/store/category`)
+  //     .then(res => res.json())
+  //     .then(categoryList => setCategoryList(categoryList.category_lists));
+  // }, []);
+
+  return (
+    <HiddenDiv onMouseLeave={props.handleCategoryOff}>
```

Comment:
```suggestion
    <Container onMouseLeave={props.handleCategoryOff}>
```
hiddenDiv라는 이름은 조금 어색해보입니다
- hidden은 컴포넌트의 상태로서 다루어져야 할 부분이고 (ex. 스타일드 컴포넌트의 props 속성을 isHidden으로 쓴다든가)
- Div는 태그 이름일 뿐이네요

저는 가장 바깥쪽의 스타일드 컴포넌트는 크게 고민 없이 Container나 Wrapper로 쓰는 편입니다

Reply:
"SubMenuContainer "로 수정하였습니다!